src/pyhf/infer/mle.py

```diff
@@ -3,23 +3,61 @@
 
 
 def twice_nll(pars, data, pdf):
-    """
-    Twice the negative Log-Likelihood.
+    r"""
+    Two times the negative log-likelihood of the model parameters, :math:`\left(\mu, \boldsymbol{\theta}\right)`, given the observed data.
```

Comment:
should we mention this is to maximize the likelihood by minimizing the objective function?

Reply:
Hm. We mention below that

```
It serves as the objective function to minimize in :func:`~pyhf.infer.mle.fit` and :func:`~pyhf.infer.mle.fixed_poi_fit`.
```

and those methods' docstrings make it clear that the likelihood is being maximized, so it feels unnecessary to me. What are your thoughts here?